wasp/boards/pinetime/watch.py.in

```diff
@@ -39,6 +40,7 @@ class Backlight(object):
     lo = Pin("BL_LO", Pin.OUT, value=0)
     mid = Pin("BL_MID", Pin.OUT, value=1)
     hi = Pin("BL_HI", Pin.OUT, value=1)
+    pwm = PWM(0, pin=Pin("BL_LO", Pin.OUT), freq=PWM.FREQ_16MHZ, duty=85, period=160000)
```

Comment:
I'm not quite sure I understood what your comment describing the problems with this. Did you mean that the backlight brightness code and the vibration motor code are interfering with each other?

If so that is probably because the motor is also using PWM channel 0 (e.g. they are programming the same PWM to send different things to different pins). I've not checked how many PWM channels nRF51 (and the micropython port) support but try setting the first argument to 1 and seeing what happens...